plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryType.java

```diff
@@ -304,15 +310,18 @@ private static String quote(String value)
         return "'" + value + "'";
     }
 
-    String convertToString(Type type, Object value)
+    public Optional<String> convertToString(Type type, Object value)
     {
+        if (this == GEOGRAPHY || type instanceof ArrayType || type instanceof RowType) {
+            return Optional.empty();
```

Comment:
- what is the enum value for Array and Row. Is it RECORD?
- i would let inidivudla enum option's toStringconverter handle the logic, by returning Optional

Reply:
> i would let inidivudla enum option's toStringconverter handle the logic, by returning Optional


i pushed a fixup with some proposed changes, but up to you whether you take them or discard

Reply:
> what is the enum value for Array and Row. Is it RECORD?

Row is RECORD, but field type is used in case of Array. e.g. `Array<boolean>` -> BOOLEAN. 
So, we can't remove the `if (type instanceof ArrayType)` until fixing it.